qcodes/instrument_drivers/AimTTi/AimTTi_PL601P_channels.py

```diff
@@ -0,0 +1,311 @@
+        self.write(f'INCI{channel_id}')
+         # Clear the cache.
+        _ = self._get_current_value()
+
+    def decrement_curr_by_step_size(self) -> None:
+        """
+        A bound method that decreases the current output of the corresponding
+        channel by an amount the step size set by the user via ``curr_step_size``
+        parameter.
+        """
+        channel_id = self.channel
+        self.write(f'DECI{channel_id}')
+         # Clear the cache.
+        _ = self._get_current_value()
+
+    def save_setup(self, slote: int) -> None:
+        """
+        A bound function that saves the output setup to the internal
+        store specified by the numbers 0-9.
+        """
+        if not slote in self.set_up_store_slots:
+            raise RuntimeError("Slote number should be an integer between"
+                               "0 adn 9.")
+
+        channel_id = self.channel
+        self.write(f'SAV{channel_id} {slote}')
+
+    def load_setup(self, slote: int) -> None:
+        """
+        A bound function that loadss the output setup from the internal
+        store specified by the numbers 0-9.
+        """
+        if not slote in self.set_up_store_slots:
+            raise RuntimeError("Slote number should be an integer between"
+                               "0 adn 9.")
+
+        channel_id = self.channel
+        self.write(f'RCL{channel_id} {slote}')
+        # Update snapshot after load.
+        _ = self.snapshot(update=True)
+
+    def set_damping(self, val: int=0) -> None:
+        """
+        Sets the current meter measurement averaging on and off.
+        """
+        if not val in [0, 1]:
+            raise RuntimeError("To 'turn on' and 'turn off' the averaging, "
+                               "use '1' and '0', respectively.")
+        channel_id = self.channel
+        self.write(f'DAMPING{channel_id} {val}')
+
+class AimTTi(VisaInstrument):
+    """
+    This is the QCoDeS driver for the Aim TTi PL-P series power supply.
+    Tested with Aim TTi PL601-P equipped with a single output channel.
+    """
+    def __init__(self, name: str, address: str, **kwargs) -> None:
+        """
+        Args:
+            name: Name to use internally in QCoDeS.
+            address: VISA resource address
+        """
+        super().__init__(name, address, terminator='\n', **kwargs)
+
+        channels = ChannelList(self, "Channels", AimTTiChannel,
+                               snapshotable=False)
+
+        _model = self.get_idn()['model']
+
+        _numOutputChannels = {'PL068-P': 1, 'PL155-P': 1, 'PL303-P': 1,
+                             'PL601-P': 1, 'PL303QMD-P': 2, 'PL303QMT': 3}
+
+        if not _model in _numOutputChannels.keys():
+            raise NotKnownModel("Unknown model, connection cannot be "
+                                "established.")
+
+        if _model is None:
+            raise RuntimeError("Model name cannot be empty "
+                               "and should be string.")
```

Comment:
I don't think this error message is very helpful to the user (they didn't provide the model name). Under which circumstances would this error be encountered? Perhaps a `_model` of `None` should also raise `NotKnownModel`.